release/docker/docker-entrypoint.sh

```diff
@@ -11,7 +12,7 @@ MITMPROXY_PATH="/home/mitmproxy/.mitmproxy"
 if [[ "$1" = "mitmdump" || "$1" = "mitmproxy" || "$1" = "mitmweb" ]]; then
   mkdir -p "$MITMPROXY_PATH"
   chown -R mitmproxy:mitmproxy "$MITMPROXY_PATH"
-  su-exec mitmproxy "$@"
+  su - mitmproxy -c "$@" # su-exec didn't have a package for Debian Buster, hence su
```

Comment:
please remove the comment - nobody is remembering that a previous version was using su-exec, or for what reason.